src/components/utils/include/utils/timer.h

```diff
@@ -43,10 +43,16 @@ typedef sigval_t sigval;
 #endif
 
 #include <time.h>
+#include <stdint.h>
```

Comment:
@AGritsevich I was right because `<cstdint>` is c++11 . We are still not allowed to use it. So @anosach-luxoft is wrong.